src/applications/static-pages/download-1095b/mocks/server.js

```diff
@@ -21,10 +22,15 @@ const responses = {
       availableForms: [{ year: 2024, lastUpdated: '2025-02-03T18:50:40.548Z' }],
     });
   },
+  // Uncomment this and import statement to mock a 500 error
+  // 'GET /v0/form1095_bs/available_forms': (_req, res) => {
+  //   return res.status(500).json(error500);
+  // },
+
   'GET /v0/user': (_req, res) => {
     // example user data cases
-    // return res.json(user.loa3User72); // default user LOA3 w/id.me (success)
-    return res.json(user.loa1User); // LOA1 user w/id.me
+    return res.json(user.loa3User72); // default user LOA3 w/id.me (success)
```

Comment:
Let's default to the LOA4 user in our mock server since that will produce the best result for viewing the app locally.

Reply:
I hope you meant LOA3 because I'm not ready for LOA4 yet.